src/config.ts

```diff
@@ -1,17 +1,16 @@
-/* tslint:disable:no-magic-numbers */
 import { config } from 'dotenv';
-config();
 
 import getBooleanOption from 'jscommons/dist/config/getBooleanOption';
 import getNumberOption from 'jscommons/dist/config/getNumberOption';
 import getStringOption from 'jscommons/dist/config/getStringOption';
 import { defaultTo } from 'lodash';
 import {
-  MongoClientOptions,
+  type MongoClientOptions,
```

Comment:
👁️  👁️  I didn't know this syntax. Should we prefer this to having the type imports separate e.g.:
```TypeScript
import type { MongoClientOptions, ReadPreferenceMode } from 'mongodb';
import { ReadPreference } from 'mongodb';
```

Reply:
@ToranSharma , hmm, not sure 😶‍🌫️ Which one do you prefer more?

I like both but separate type import looks more readable 🤔

Reply:
🤷 Yeah I don't know. When we move this into enterprise, we should make it consistent. Right now it is fine I guess.